metrics/sample_test.go

```diff
@@ -242,15 +243,15 @@ func TestUniformSampleIncludesTail(t *testing.T) {
 	}
 }
 
-func TestUniformSampleSnapshot(t *testing.T) {
-	s := NewUniformSample(100)
-	for i := 1; i <= 10000; i++ {
-		s.Update(int64(i))
-	}
-	snapshot := s.Snapshot()
-	s.Update(1)
-	testUniformSampleStatistics(t, snapshot)
-}
+// func TestUniformSampleSnapshot(t *testing.T) {
+// 	s := NewUniformSample(100)
+// 	for i := 1; i <= 10000; i++ {
+// 		s.Update(int64(i))
+// 	}
+// 	snapshot := s.Snapshot()
+// 	s.Update(1)
+// 	testUniformSampleStatistics(t, snapshot)
+// }
```

Comment:
Are these meant to stay commented ?

Reply:
Made a another try, thankfully it worked.